test/python/WMCore_t/REST_t/Api_t.py

```diff
@@ -225,9 +225,23 @@ def test_multi_throw10(self):
 
 def setup_server():
     srcfile = __file__.split("/")[-1].split(".py")[0]
-    setup_test_server(srcfile, "Root")
-    #cpconfig.update({"log.screen": True})
+    server, T.test_authz_key = setup_test_server(srcfile, "Root")
+    print("SERVER", server.config.pythonise_())
+    print("AUTHZ", T.test_authz_key, T.test_authz_key.data)
+
+import cherrypy
+from cherrypy.test import webtest
```

Comment:
Yes, I know I'll make everything clean once I'll see results. I also
plant to re-factor a little bit and move this section into module which will be
used by other tests. That's why I kept them together (the imports and
underneath code which will be moved).

On  0, Eric Vaandering <notifications@github.com> wrote:
> > @@ -225,9 +225,23 @@ def test_multi_throw10(self):
> >  
> >  def setup_server():
> >      srcfile = __file__.split("/")[-1].split(".py")[0]
> > -    setup_test_server(srcfile, "Root")
> > -    #cpconfig.update({"log.screen": True})
> > +    server, T.test_authz_key = setup_test_server(srcfile, "Root")
> > +    print("SERVER", server.config.pythonise_())
> > +    print("AUTHZ", T.test_authz_key, T.test_authz_key.data)
> > +
> > +import cherrypy
> > +from cherrypy.test import webtest
> 
> Let's keep imports at the top unless this is just a test Valentin is trying to run (not really asking to merge)
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6183/files#r38317754